qiskit_ibm_provider/ibm_backend.py

```diff
@@ -490,6 +490,14 @@ def run(
             # Transpiling in circuit-runner is deprecated.
             run_config_dict["skip_transpilation"] = True
 
+        if isinstance(circuits, (QuantumCircuit, Schedule)):
+            circuits = [circuits]
+
+        if run_config_dict["skip_transpilation"]:
```

Comment:
I don't think you need this check. The `circuit-runner` **always** skips transpilation, no?

Reply:
_Technically_ you can still set it to false for `circuit-runner` (and `qasm3-runner`), since we only deprecated it (in the programs) to keep backward compatibility.